shelley/chain-and-ledger/executable-spec/src/Cardano/Ledger/Shelley/API/NonMyopic.hs

```diff
@@ -0,0 +1,58 @@
+module Cardano.Ledger.Shelley.API.NonMyopic
```

Comment:
[bikeshedding] `NonMyopic` is a somewhat confusing name here... how about something like `Wallet`, since this is exposing functionality for the wallet

Reply:
sounds good to me!